src/models/proficiency.ts

```diff
@@ -0,0 +1,40 @@
+import * as mongoose from 'mongoose';
+import { APIReference, APIReferenceSchema } from './common';
+
+interface Reference {
+  index: string;
+  name: string;
+  type: string;
+  url: string;
+}
+
+const Reference = {
+  index: { type: String, index: true },
+  name: { type: String, index: true },
+  type: { type: String, index: true },
+  url: { type: String, index: true },
+};
+
+interface Proficiency {
+  _id?: mongoose.Types.ObjectId;
+  classes: APIReference[];
+  index: string;
+  name: string;
+  races: APIReference[];
+  reference: Reference;
```

Comment:
This is not an array?

Reply:
So this is probably a bit of a bug I guess?
The API returns a list, so my tests were failing against this. Looks like this may not be working in the JS side either.
But you are correct, it seems it should not be a list!